src/managers/balancer.py

```diff
@@ -49,15 +49,22 @@ def storages(self) -> str:
 
     def create_internal_topics(self) -> None:
         """Create Cruise Control topics."""
+        if self.charm.state.runs_broker:
+            property_file = f'{BROKER.paths["CONF"]}/client.properties'
+            bootstrap_servers = self.charm.state.internal_bootstrap_server
+        else:
+            property_file = f'{BALANCER.paths["CONF"]}/cruisecontrol.properties'
+            bootstrap_servers = self.charm.state.balancer.broker_uris
+
```

Comment:
**question** Is this for the case where balancer is run on a different application?

Reply:
I am still trying various solutions to an authentication issue.
> charmed-kafka.topics --list --bootstrap-server 10.236.35.136:9092,10.236.35.85:9092,10.236.35.97:9092 --command-config /var/snap/charmed-kafka/current/etc/cruise-control/cruisecontrol.properties, stdout=Error while executing topic command : Authentication failed during authentication due to invalid credentials with SASL mechanism SCRAM-SHA-512

Though we might think that large deployments are more prone to errors, this specific issue only happens in single-app mode. I was able to confirm that using the `admin` or `sync` credentials works, I don't know why this fails with `balancer`.
The user does exist, the credentials are the same as in the secret, and yet I get this auth error again and again.

To add insult to the injury, this :arrow_double_up:  setup was working fine when running the integration test locally, but fails on the CI due to the `client.properties` file missing.

Currently, I have resigned myself to using the admin user to create the 3 mandatory topics

Reply:
We found the issue, will be fixed in subsequent PR